src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,51 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+interface Props {
+  selectedTodoId: number;
+  removeTodo: () => void;
+  todos: Todo[];
+}
+
+export const TodoModal: React.FC<Props> = ({
+  selectedTodoId,
+  removeTodo,
+  todos,
+}) => {
+  const [selectedTodo, setSelectedTodo] = useState<Todo | undefined>();
```

Comment:
will be better set null if you don't have value
```suggestion
  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
```